sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -110,6 +110,26 @@ public partial class ArtifactTagProperties
         public string RegistryLoginServer { get { throw null; } }
         public string RepositoryName { get { throw null; } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ContainerRegistryAudience : System.IEquatable<Azure.Containers.ContainerRegistry.ContainerRegistryAudience>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ContainerRegistryAudience(string value) { throw null; }
+        public static Azure.Containers.ContainerRegistry.ContainerRegistryAudience AzureResourceManagerChina { get { throw null; } }
```

Comment:
I originally had these as Azure<Cloud>, but Scott suggested making it clear that these are ARM scopes: https://github.com/Azure/azure-sdk-for-net/pull/23141#discussion_r684544469

I think Scott is right about this -- in the future, we want people using the ACR scopes instead of the ARM scopes.  

But I'm open to suggestions ... do you have thoughts on better names for these?

Reply:
In the context of the type name `ContainerRegistryAudience`, I was thinking that the cloud name alone would be sufficient.

Reply:
In the future, the service will support ACR scopes as well.  So customers will need to choose between the ARM cloud audiences and the ACR cloud audiences.

Reply:
Ah ok - so they would be the same name except for the context? Just curious what the actual string values of each would be.

Reply:
The string value for the ACR-scoped token hasn't been established yet, although we have requested that it be the same across clouds (like Storage does), so the customer doesn't have to specify.  The values we have for the ARM ones are the standard ARM ones from here: https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/services-support-managed-identities#azure-resource-manager